server/plugin/template.go

```diff
@@ -429,6 +435,12 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 {{- else }} unstarred
 {{- end }} by {{template "user" .GetSender}}
 It now has **{{.GetRepo.GetStargazersCount}}** stars.`))
+
+	template.Must(masterTemplate.New("newReleaseEvent").Funcs(funcMap).Parse(`
+{{template "repo" .GetRepo}} {{template "user" .GetSender}}
+{{- if eq .GetAction "created" }} created a release {{template "release" .GetRelease}}
+{{- else if eq .GetAction "deleted" }} deleted a release {{template "release" .GetRelease}}
```

Comment:
Are there any other actions besides these two?

Reply:
Yes the other events are:
- Release created
- Release edited
- Release published
- Release unpublished
- Release deleted